mmaction/core/evaluation/eval_hooks.py

```diff
@@ -60,6 +63,7 @@ def __init__(self,
                  by_epoch=True,
                  save_best='auto',
                  rule=None,
+                 out_dir=None,
```

Comment:
How about saving it to work_dir, like the ckpts?

Reply:
Also vote for saving to the `work_dir`

Reply:
An advice: you can set the default value of `out_dir` as `work_dir`. For example, if  `out_dir is None`, you output the eval results to `work_dir`.

Reply:
It's OK to save eval results by default, since it doesn't take much storage space.

Reply:
Okay